include/envoy/router/router_ratelimit.h

```diff
@@ -69,6 +69,11 @@ class RateLimitPolicy {
   virtual ~RateLimitPolicy() {}
 
   /**
+   * @return true if there is no rate limit policy.
+   */
+  virtual bool empty() const PURE;
```

Comment:
Do we really need this? Can we just call getApplicableRateLimit() and see if empty is returned?